README.md

```diff
@@ -240,10 +240,37 @@ License: By requesting and using the Container OS Image for Windows containers,
 
 ### Telemetry
 
-By default, PowerShell collects the OS description and the version of PowerShell (equivalent to `$PSVersionTable.OS` and `$PSVersionTable.GitCommitId`) using [Application Insights](https://azure.microsoft.com/services/application-insights/).
-To opt-out of sending telemetry, create an environment variable called `POWERSHELL_TELEMETRY_OPTOUT` set to a value of `1` before starting PowerShell from the installed location.
+Sending telemetry is controlled via an environment variable called `POWERSHELL_TELEMETRY_OPTOUT`.
+Set the value to `1`, `yes`, or `true` before starting PowerShell to disable telemetry.
+Setting the value to `0`, `no`, `false`, or not creating the variable will enable telemetry.
 The telemetry we collect falls under the [Microsoft Privacy Statement](https://privacy.microsoft.com/privacystatement/).
 
+The following is sent when the shell starts:
+
+- Operating system information
+- The commit id of the build
+- The enabled experimental features
+- A randomly generated guid representing the session
+- A randomly generated guid representing the system
+- Geo-location information is supplied by Application Insights (Country, Region, State, City, if available)
+- The value of the environment variable `POWERSHELL_DISTRIBUTION_CHANNEL`
+
+During shell operation the following is sent periodically:
+
+- The name of a module when loaded if it is in the pre-approved list of modules (see line 128 in src/System.Management.Automation/utils/Telemetry.cs).
+Modules which are not in this predetermined list are reported as "Anonymous".
+- The type of item being executed (Alias, Function, Filter, Cmdlet, ExternalScript, Application, Script, or Configuration).
```

Comment:
Just out of curiosity, how useful do we think it is to collect the command type information?

Reply:
@JamesWTruher can you please comment on this question? I'm just curious how useful this information turns out to be in our telemetry analysis so far.